### ISSUE TITLE
HTML embedded not working

### ISSUE THREAD

**Issue by elielberra** [RocketChat/Rocket.Chat#25989]
Last week we were able to embed html code into the messages, right now this feature is gone. It was a very useful feature, is it possible to bring it back? Thank you

**Comment by thrakks** [comment]
+1 to this. My company uses the ability to embed html code in order to display data in tables etc. via an integration bot. They moved to this way of displaying data after Attachments with more than 2 columns were broken. I'll never get the users to switch back or to an as yet undiscovered third way of displaying data.

PLEASE allow html embedded as an optional toggle knowing full well its a security risk that's on the company choosing to to enable it.

**Comment by Avalarion** [comment]
I would agree on this. We have had our vacation calendar posted once a week...
